src/index.js

```diff
@@ -291,19 +290,6 @@ function createConfig(options, entry, format, writeMeta) {
 	let nameCache = {};
 	let mangleOptions = options.pkg.mangle || false;
 
-	let exportType;
-	if (format !== 'es') {
```

Comment:
Could we make it an option instead of deleting?

Reply:
We could put this code behind an option. In either case it needs to be disabled for TypeScript because this breaks the generated typing definitions.